lib/utils/noFilesFoundError.js

```diff
@@ -11,9 +13,33 @@ class NoFilesFoundError extends Error {
 			fileList = [fileList];
 		}
 
-		const pattern = fileList.filter((i) => !i.startsWith('!')).join(', ');
+		const filteredFileList = fileList.filter((i) => !i.startsWith('!'));
+		const pattern = filteredFileList.join(', ');
 
 		this.message = `No files matching the pattern "${pattern}" were found.`;
+
+		if (
+			os.platform() === 'win32' &&
+			filteredFileList.filter((file) => file.match(/^'.*'$/)).length > 0
+		) {
+			const packageScript = process.env.npm_lifecycle_event || '';
```

Comment:
What about yarn?

Reply:
Hey @evilebottnawi, `yarn` also uses the same env var. Good point though, are there any other package managers we should consider, that perhaps don't adhere to the same spec? I've only ever used `yarn` and `npm` myself.